packages/web/src/new/utils/analytics/amplitude.ts

```diff
@@ -0,0 +1,28 @@
+import { init, track } from '@amplitude/analytics-browser';
+import assert from 'ts-invariant';
+
+import type { AmplitudeActions, ComplexActions } from 'new/utils/analytics/types';
+
+export const initAmplitude = () => {
```

Comment:
This function is old fault i think. We can just remove the function definition and init amplitude in "body" of this file.